packages/agoric-cli/src/commands/inter.js

```diff
@@ -83,56 +141,144 @@ export const fmtBid = (bid, assets) => {
 
   const {
     id,
-    error,
     proposal: { give },
     offerArgs: { want },
     payouts,
+    result,
+    error,
   } = bid;
+  const resultProp =
+    !error && result && result !== 'UNPUBLISHED' ? { result } : {};
   const props = {
     ...(give ? { give: fmt.record(give) } : {}),
     ...(want ? { want: fmt.amount(want) } : {}),
-    ...(payouts ? { payouts: fmt.record(payouts) } : {}),
+    ...(payouts ? { payouts: fmt.record(payouts) } : resultProp),
     ...(error ? { error } : {}),
   };
   return harden({ id, ...spec, ...props });
 };
 
 /**
+ * Make Inter Protocol liquidation bidding commands.
+ *
  * @param {{
  *   env: Partial<Record<string, string>>,
  *   stdout: Pick<import('stream').Writable,'write'>,
  *   stderr: Pick<import('stream').Writable,'write'>,
  *   now: () => number,
  *   createCommand: // Note: includes access to process.stdout, .stderr, .exit
  *     typeof import('commander').createCommand,
- *   execFileSync: typeof import('child_process').execFileSync
+ *   execFileSync: typeof import('child_process').execFileSync,
+ *   setTimeout: typeof setTimeout,
  * }} process
  * @param {{ fetch: typeof window.fetch }} net
  */
 export const makeInterCommand = async (
-  { env, stdout, stderr, now, execFileSync, createCommand },
+  {
+    env,
+    stdout,
+    stderr,
+    now,
+    setTimeout,
+    execFileSync: rawExec,
+    createCommand,
+  },
   { fetch },
 ) => {
   const interCmd = createCommand('inter')
-    .description('Inter Protocol tool')
-    .option('--home [dir]', 'agd CosmosSDK application home directory')
+    .description('Inter Protocol commands for liquidation bidding etc.')
+    .option('--home <dir>', 'agd CosmosSDK application home directory')
```

Comment:
we'll want this flexiblity for all agops but I don't expect this PR to update all the commands. `bin/agops inter` a command set rather than a single command so it can be different from other agops